message/html/markdown.js

```diff
@@ -42,7 +42,7 @@ exports.create = function (api) {
           return renderEmoji(emoji, url)
         },
         toUrl: (id) => {
-          if (ref.isBlob(id)) return api.blob.sync.url(id)
+          if (id[0] == '&') return api.blob.sync.url(id)
```

Comment:
Unrelated change?

Reply:
oops, yeah that should be part of a different PR https://github.com/ssbc/patchcore/pull/50